apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/TransformerContext.java

```diff
@@ -0,0 +1,44 @@
+package gov.cms.bfd.server.war.commons;
+
+import com.codahale.metrics.MetricRegistry;
+import java.util.Optional;
+
+/**
+ * Contains contextual information needed for transformation including configuration, metrics
+ * registry and drug code display lookup.
+ */
+public class TransformerContext {
+
+  private final MetricRegistry metricRegistry;
+  private final Optional<Boolean> includeTaxNumbers;
+  private final FdaDrugCodeDisplayLookup drugCodeDisplayLookup;
+
+  /**
+   * @param metricRegistry the {@link MetricRegistry} to use
+   * @param includeTaxNumbers the {@link Optional} populated with an {@link Boolean} to use
+   * @param drugCodeDisplayLookup the {@link FdaDrugCodeDisplayLookup} to use
+   */
+  public TransformerContext(
+      MetricRegistry metricRegistry,
+      Optional<Boolean> includeTaxNumbers,
+      FdaDrugCodeDisplayLookup drugCodeDisplayLookup) {
+    this.metricRegistry = metricRegistry;
+    this.includeTaxNumbers = includeTaxNumbers;
+    this.drugCodeDisplayLookup = drugCodeDisplayLookup;
+  }
+
+  /** @return the short identifier for this {@link MetricRegistry} */
+  public MetricRegistry getMetricRegistry() {
+    return metricRegistry;
+  }
+
+  /** @return the short identifier for this {@link Optional} populated with an {@link Boolean} */
+  public Optional<Boolean> getIncludeTaxNumbers() {
+    return includeTaxNumbers;
+  }
+
+  /** @return the short identifier for this {@link FdaDrugCodeDisplayLookup} */
```

Comment:
rather than referencing the short identifier, can we simply use `@return the {@link drugCodeDisplayLookup}` in these three getters?